src/test/java/study/StringTest.java

```diff
@@ -0,0 +1,34 @@
+package study;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.AssertionsForClassTypes.*;
+
+public class StringTest {
+    @Test
+    void split() {
+        String[] result = "1,2".split(",");
+        String[] result2 = "1".split(",");
+        assertThat(result).contains("1","2");
+        assertThat(result).containsExactly("1","2");
+        assertThat(result2).contains("1");
```

Comment:
```suggestion
        String[] result2 = "1".split(",");
        assertThat(result2).contains("1");
```

사용하는 변수를 가능한 가까운 곳에 구현하는 것은 어떨까?